### PR TITLE
Edited chapter 3.1.5 Transport Controls.

### PR DESCRIPTION
**Pull request by deborahtrez**
In reference to this issue, https://github.com/mixxxdj/manual/issues/124
I removed the Fast-Forward/-Rewind buttons as suggested by @ronso0  because they are not used in skins anymore. 
I commented out the .rst for that section.

@Holzhaus @ywwg waiting for your approval...

### CODE REVIEW

**Comment by ronso0** [comment]
add to whitelist

**Comment by ronso0** [comment]
Unfortunately I don't agree with my own proposal anymore ;)
In a recent skin redesign we decided to keep the Reverse/reverseroll button in LateNight skin
image

Right now, I think we should go with the second proposal: keep the section but mention that not all controls maybe present in all skins anymore.

**Comment by deborahtrez** [comment]
Okay. I will make the changes.

Are the blue buttons in that screenshot part of the Transport controls too? If, yes, can i add a description for each in the manual as well? (What do they do?)

> Unfortunately I don't agree with my own proposal anymore ;)
> In a recent skin redesign we decided to keep the Reverse/reverseroll button in LateNight skin
> image
> 
> Right now, I think we should go with the second proposal: keep the section but mention that not all controls maybe present in all skins anymore.
> We could add a comment that those controls can be mapped to keyboards and MIDI controllers, though, and link to https://github.com/mixxxdj/mixxx/wiki/mixxxcontrols.
> I'm not sure how to include the actual control names here without getting to technical.

**Comment by ronso0** [comment]
I'm currently going through the manual and list all spots that need updates.
Those blue buttons are special cue points to mark the intro & outro of a track.

Did you consider installing Mixxx and test it yourself to get a live impression?

**Comment by ronso0** [comment]
> I'm currently going through the manual and list all spots that need updates.
Those blue buttons are special cue points to mark the intro & outro of a track.

But those are not present in 2.2.x yet, they were added in 2.3

**Comment by Holzhaus** [comment]
Yeah, we need to keep an eye on the branch each PR is going into. I don't want to accidently merge Screenshots and information for Mixxx 2.3 into the `manual-2.2.x` branch.

**Comment by deborahtrez** [comment]
> I'm currently going through the manual and list all spots that need updates.
> Those blue buttons are special cue points to mark the intro & outro of a track.
> 
> Did you consider installing Mixxx and test it yourself to get a live impression?

Yes. I tried that but i got this error. 

image

Im not sure why but i do have Microsoft Visual C++ 2015 Redistributable on my computer.

image


So i tried downloading the beta version of the Mixxx software instead, so i could see what new features are being worked upon.
The download button for Mixxx 2.3.0 (Beta) Windows does not work. (Nothing gets downloaded). I used this [link](https://www.mixxx.org/download/#stable)

I'm still trying to find a solution to the first error. Then i will revert to this issue. Any suggestions?

**Comment by ronso0** [comment]
The error is known. For 2.2.x there's a workaround described in https://bugs.launchpad.net/mixxx/+bug/1814273

Meanwhile, you can try the ` mixxx-2.3.0-alpha-pre-master-release-x86-latest.exe` or ` mixxx-2.3.0-alpha-pre-master-release-x64-latest.exe` from http://downloads.mixxx.org/builds/master/release/

If you run into the bug with that version, too, have a look at the solution posted in https://bugs.launchpad.net/mixxx/+bug/1862795

**Comment by deborahtrez** [comment]
Hi @ronso0 I was able to fix my errors. I have the Mixxx software installed and working perfectly. This is what my interface looks like. The version that i have is 2.2.4

image

Going through the documentation, i noticed that the interface that i have looks different from what is described in the documentation. For example, In the documentation, this is what is shown for the Playback/Cue controls,

image

This is what i have on mine.

image

The version that is described in the manuals is for Mixxx Version 2.2.3.
Would you like me to update the manuals for Version 2.2.4? (this is the version from which the Transport Controls were removed)

**Comment by deborahtrez** [comment]
> Unfortunately I don't agree with my own proposal anymore ;) Sorry
> The point is, in a recent skin redesign we decided to keep the Reverse/reverseroll button in LateNight skin
> image
> 
> Right now, I think we should go with the second proposal: keep the section but mention that not all controls maybe present in all skins anymore.
> We could add a comment that those controls can be mapped to keyboards and MIDI controllers, though, and link to https://github.com/mixxxdj/mixxx/wiki/mixxxcontrols.
> I'm not sure how to include the actual control names here without getting to technical.

Let me do this.

**Comment by deborahtrez** [comment]
I have added a note at the bottom of the Transport Controls section to implement this [change.](https://github.com/mixxxdj/manual/pull/143#issuecomment-706568943)
@ronso0  Waiting for your approval....

**Comment by Holzhaus** [comment]
As you noticed, the Screenshots are inconsistent in the manual. Some screenshots are old and are from Mixxx 1.10. Also, Mixxx supports different skins (have a look at https://mixxx.org/screenshots/ and scroll down to Mixxx 2.2).

In Mixxx 2.2, "Deere" is the default skin. In Mixxx 2.3, we changed the default skin to an updated version of "LateNight" skin.

The manual should always use screenshots of the default skin for that version. So yeah, feel free to go ahead and update the screenshots and use use the 2.2 Deere skin throughout the manual. For the `manual-2.3.x` branch, we need to use 2.3 version of "LateNight" with the "PaleMoon" color scheme instead, but maybe it's worth waiting for that until https://github.com/mixxxdj/mixxx/pull/3145 has been merged.

**Comment by deborahtrez** [comment]
> As you noticed, the Screenshots are inconsistent in the manual. Some screenshots are old and are from Mixxx 1.10. Also, Mixxx supports different skins (have a look at https://mixxx.org/screenshots/ and scroll down to Mixxx 2.2).
> 
> In Mixxx 2.2, "Deere" is the default skin. In Mixxx 2.3, we changed the default skin to an updated version of "LateNight" skin.
> 
> The manual should always use screenshots of the default skin for that version. So yeah, feel free to go ahead and update the screenshots and use use the 2.2 Deere skin throughout the manual. For the `manual-2.3.x` branch, we need to use 2.3 version of "LateNight" with the "PaleMoon" color scheme instead, but maybe it's worth waiting for that until [mixxxdj/mixxx#3145](https://github.com/mixxxdj/mixxx/pull/3145) has been merged.

Alright. I'm happy to do this. I think it'd be better to create a new issue for this since the change will affect more chapters throughout the manual. Let me get started on it.

File: source/chapters/user_interface.rst
Comment (Holzhaus) [inline source/chapters/user_interface.rst:373]: Please undo this, I think this was probably changed by accident.

File: source/chapters/user_interface.rst
Comment (Holzhaus) [inline source/chapters/user_interface.rst:407]: Let's add an empty line after the note. 

```suggestion
 .. note:: Not all controls may be present in all skins anymore. However, these controls can be mapped to keyboards and MIDI controllers. See this `guide. <https://github.com/mixxxdj/mixxx/wiki/mixxxcontrols>`_

```

**Review by Holzhaus — CHANGES_REQUESTED** [review]
Thank you.

File: source/chapters/user_interface.rst
Comment (deborahtrez) [reply source/chapters/user_interface.rst:373]: Oh, i'm sorry about this. Change undone.

File: source/chapters/user_interface.rst
Comment (deborahtrez) [reply source/chapters/user_interface.rst:407]: Empty line added!

**Comment by deborahtrez** [comment]
Hey @Holzhaus . Requested changes have been effected. Ready for your review...

**Review by Holzhaus — APPROVED** [review]

**Comment by deborahtrez** [comment]
> @deborahtrez Changes LGTM but unfortunately the propsed link below leeds to the wiki which says controls were moved to the manual but the controls are only in the 2.3 version.
> @Holzhaus we can either restore the wiki page and leave the Manual 2.3 link at the top, or you are willing to backport the controls to the 2.2 manual? 😉

Hey @ronso0 . I hope you're doing okay. How do you propose we rectify this? I could change the statement to say that the controls can be mapped on to the skin, but this is only available for version 2.3...?
Or...since the controls aren't in the 2.2.x skins anymore, we could omit that part from the manual.....
I would love to get your feedback..

**Comment by Holzhaus** [comment]
I'm not a fan of having to maintain the Mixxx control wiki page in addition to the manual. Changes made to that wiki page will probably be lost and it doesn't help because the page documents 2.3-only controls anyway. 

I think it's okay to keep the link in the 2.2 manual to the wiki page and direct people back to the 2.3 manual.

**Comment by ronso0** [comment]
Okay then. I added **Mixxx 2.3** to the link in that wiki page.

**Comment by ronso0** [comment]
LGTM, thank you!

**Comment by deborahtrez** [comment]
> LGTM, thank you!

You're welcome. Thank You.

**Pull request merged**
